portal/db.py

```diff
@@ -85,4 +88,31 @@ def init_app(app):
     app.teardown_appcontext(close_db)
     app.cli.add_command(init_db_command)
     app.cli.add_command(mock_db_command)
+    app.cli.add_command(import_csv_command)
+
+
+def import_csv():
+    """Seed the database with mock data."""
 
+    data_filepath = os.path.join(os.path.dirname(
+        __file__), os.pardir, "import_data", "portal_users.csv")
+    # open the mock data file and close when done
+    with open(data_filepath, "r") as f:
+        with get_db() as con:
+            with con.cursor() as cur:
+                reader = csv.reader(f)
+                next(reader)  # Skip the header row.
+                for row in reader:
+                    cur.execute(
+                        "INSERT INTO users VALUES (%s, %s, %s, %s, %s, %s)",
+                        row
+                    )
```

Comment:
At some point, your `users` table might have more columns than this CSV provides so you should be explicit about what columns you are inserting:
```sql
INSERT INTO users (col1, col2, ...)
VALUES (%s, %s, ...)
```